docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -58,8 +58,8 @@ class Employee {
         this.surname = surname;
     }
 
-    toString() {
-        return "Employee: " + this.surname;
+    toHazelcastJsonValue() {
```

Comment:
I can't add comment to that line so, putting it here: 

In a bit below the code samples, we show a log like 
```
entryAdded:EntryEvent {Address[192.168.178.10]:5702} key=1,oldValue=null,
value=Person{surname= smith }, event=ADDED, by Member [192.168.178.10]:5702
```
which is not in line with the current status

Reply:
Updated in 27094ff09f1fe121a6d845c0c40fc96177664343